gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -337,14 +310,15 @@ public static ZoneListOption pageSize(int pageSize) {
      * can be used.
      */
     public static ProjectOption fields(ProjectField... fields) {
-      return new ProjectOption(DnsRpc.Option.FIELDS, ProjectField.selector(fields));
+      return new ProjectOption(DnsRpc.Option.FIELDS,
+          Helper.selector(ProjectField.REQUIRED_FIELDS, fields));
     }
   }
 
   /**
    * Class for specifying change request field options.
    */
-  class ChangeRequestOption extends AbstractOption implements Serializable {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://www.codacy.com/app/mziccard/gcloud-java/file/2331164897/issues/source?bid=3123103&fileBranchId=3204660#l321)